horace_core/GUI/horace.m

```diff
@@ -49,6 +49,24 @@
     strcmp(element_class,'d3d') || strcmp(element_class,'d4d') ||...
     strcmp(element_class,'sqw');
 
+function [vec_val,ok] = read_vector(str_val)
```

Comment:
Why is a new function returning `ok` rather than using error handling?

Reply:
I did minimal changes to what was there and this is easiest way of achieving this. ~ok will lead to different error messages depending on what vector is read.  Error handler is warped around each call to this operation and lead to different error messages to GUI -- changing this is too much for me to handle at this stage.

Reply:
Try-catches can be easily nested and `rethrow` means if not handled pass up to the next handler, it would be no different to the current if structure, but allows `read_vector` to be potentially portable, rather than limited in scope.